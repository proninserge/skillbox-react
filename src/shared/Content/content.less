@import '~@/variables.global.less';

.content {
    @media all and (min-width: 1024px) {
        & {
            background-color: @white;
            border-radius: 7px 7px 0 0;
        }        
    }
}