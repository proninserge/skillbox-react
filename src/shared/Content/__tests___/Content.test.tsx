import {shallow, configure} from 'enzyme';
import {Content} from '../Content';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

describe('Content', () => {
    const wrapper = shallow(<Content children={[<div key="poi" className="child-one"/>, <div key="uyt" className="child-two"/>]} />);

    test('should render with children', () => {
        expect(wrapper).toBeDefined();
        expect(wrapper.find('main.content').isEmptyRender()).toBeFalsy();
        expect(wrapper.find('div.child-one').isEmptyRender()).toBeFalsy();
        expect(wrapper.find('div.child-two').isEmptyRender()).toBeFalsy();
    });

    test ('should render snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

});