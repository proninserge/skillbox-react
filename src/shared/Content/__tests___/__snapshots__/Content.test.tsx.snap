// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render snapshot 1`] = `
<main
  className="content"
>
  <div
    className="child-one"
    key="poi"
  />
  <div
    className="child-two"
    key="uyt"
  />
</main>
`;
